java/ql/lib/semmle/code/java/frameworks/kotlin/StdLibGenerated.qll

```diff
@@ -0,0 +1,1875 @@
+/**
+ * THIS FILE IS AN AUTO-GENERATED MODELS AS DATA FILE. DO NOT EDIT.
+ * Definitions of taint steps in the StdLibGenerated framework.
+ */
```

Comment:
Is it possible to add the Kotlin version number to this header?

Reply:
Done in https://github.com/github/codeql/pull/9783/commits/d07effb202876c6e5053e6eae5631d1d43485655.